cmd/root.go

```diff
@@ -197,6 +197,24 @@ func initConfig() {
 	viper.BindEnv("OIDC_SUBJECT_TYPE_PAIRWISE_SALT")
 	viper.SetDefault("OIDC_SUBJECT_TYPE_PAIRWISE_SALT", "public")
 
+	viper.BindEnv("TRACING_PROVIDER")
+	viper.SetDefault("TRACING_PROVIDER", "")
+
+	viper.BindEnv("TRACING_PROVIDER_JAEGER_SAMPLING_SERVER_URL")
+	viper.SetDefault("TRACING_PROVIDER_JAEGER_SAMPLING_SERVER_URL", "http://localhost:5778/sampling")
```

Comment:
This should be empty, we can't assume that this service is running on that port and doing so could even be a security issue (it's a different service/malware not explicitly configured)